alpenhorn/util.py

```diff
@@ -111,6 +111,48 @@ def get_hostname() -> str:
     return socket.gethostname().split(".")[0]
 
 
+def pretty_deltat(seconds: float) -> str:
+    """Return a nicely formatted time delta.
+
+    Parameters
+    ----------
+    seconds : float
+        The time delta, in seconds
+
+    Returns
+    -------
+    pretty_deltat : str
+        A human-readable indication of the time delta.
+
+    Raises
+    ------
+    TypeError
+        `seconds` was non-numeric
+    ValueError
+        `seconds` was less than zero
+    """
+
+    # Reject weird stuff
+    try:
+        seconds = float(seconds)
+    except (TypeError, ValueError):
+        raise TypeError("non-numeric time delta")
+
+    if seconds < 0:
+        raise ValueError("negative time delta")
```

Comment:
Is this bad?

It's possible alpenhornd could end up having a negative time delta due to some sort of clock skew on the host.  In that case, would it be better to simply print the nonsensical delta and leave it at that, rather than crashing?

Reply:
Yeah, I think I'll remove it.